docs/src/cli-reference.rst

```diff
@@ -4,8 +4,29 @@
 
 .. _cli-reference:
 
-********************************************************************
+*************
 CLI Reference
-********************************************************************
+*************
+
+The **Tensile** project provides several command line tools. Below
+we enumerate these tools and provide links to the command line interface
+(CLI) reference documentation for each tool.
+
+1. :ref:`TensileCreateLibrary<tensilecreatelibrary-cli-reference>`
+
+.. note::
+   The following table provides the command-line syntax assumed
+   for all Tensile command-line documentation.
+
+   .. table:: Command-line syntax
+      :width: 70%
+      :align: center
+
+      ================================= ==================================
+      Notation                          Description
+      ================================= ==================================
+      ``<Text inside angle brackets>``  Required argument.
+      ``[Text inside square brackets]`` Optional arguments.
+      ``{Test inside braces}``          Set of arguments. One is required.
```

Comment:
Are the curly braces intended to include multiple optional arguments like `[OPTION ...]`? If not, we should add this row for clarity.

Reply:
{a, b, c} -> user must choose a or b or c. Suggestions?

Reply:
Makes sense, that sounds good to me